src/pages/ResultPage/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { useHistory } from "react-router";
+import { useQuizData } from "../../context/quizContext";
+import { AnswerObject } from "../../types/quiz.types";
+import { CardWrapper, QuestionText } from "../../components/QuizCard/styles";
+import { StartButton } from "../../components/CategoryCard/styles";
```

Comment:
You can also export `styles.ts` from `index.tsx` in `components` folder

Reply:
done